scheduler/scheduler.go

```diff
@@ -189,6 +189,25 @@ func (s *Scheduler) Sync(ctx context.Context, client schema.ClientMeta, tables s
 
 	// send migrate messages first
 	for _, table := range tables.FlattenTables() {
+		if syncClient.deterministicCQID {
+			// No PK adjustment should occur if `_cq_id` is not present in the table
+			cqIDCol := table.Columns.Get(schema.CqIDColumn.Name)
+			if cqIDCol == nil || !cqIDCol.PrimaryKey {
```

Comment:
I'm missing something here about the `!cqIDCol.PrimaryKey` condition.
Shouldn't we set cqID to be the PK if it's present and not the PK? Then it will be calculated from the other PKs?

> This change updates the MigrateMessage so that only the _cq_id will be a PK and it will not have a Unique constraint on it

Basing on ⬆️ 

Is the purpose to do this here so we can remove the `pk_mode` on the destinations?

Reply:
If a user specifically generates a `_cq_id` column that is not a PK, I don't think we should override it...

Reply:
> If a user specifically generates a `_cq_id` column that is not a PK

What do you mean by that? Aren't we the ones adding the `_cq_id` column

I'll clarify, what happens when a table has 3 PKs that neither are the CQ ID column? Shouldn't we make CQ ID the PK and remove the others?

Reply:
Discussed with @hermanschaaf... the `_cq_id` is a reserved column, if it is present it should be considered a PK and we should override it.

Changed in f50a58781e03b04f4c5fa0edaf587f3691cca936